### ISSUE TITLE
ClojureScript as provided dependency

### ISSUE THREAD

**Issue by r0man** [applied-science/js-interop#17]
Hello, 

is it possible to declare ClojureScript as a provided dependency in this project? If I have shadow-cljs and js-interop as project dependencies for example, I generally want the ClojureScript version from shadow-cljs, and not the one from a library.

In Leiningen this is done by adding :scope "provided" to a dependency. I read [1] deps.edn does not support this, and the recommended way is to use aliases I think.

WDYT?

Thanks, r0man.

[1] https://ask.clojure.org/index.php/9110/scope-in-deps-edn-should-be-added-and-not-deleted-from-pom-xml

**Comment by mhuebert** [comment]
Hi @r0man, thanks for the suggestion. I have moved the clj(s) deps to a `:dev` alias and updated the tests, and released `[applied-science/js-interop "0.2.5"]`. 

Small note: the clojars coordinate for new published versions is now `applied-science/js-interop`, a slight change from before (added a hyphen) as we recently needed to do a little clean-up for consistent naming across all our applied-science projects.
